docs/integrations/microsoft-azure/kubernetes.md

```diff
@@ -260,7 +260,7 @@ Sumo Logic Metrics source is currently in Beta, to participate, contact your Sum
 
 In the Sumo Logic Azure Metrics source configuration,
```

Comment:
I think we can remove this line also `In the Sumo Logic Azure Metrics source configuration` because in the next line we are saying to refer to another docs